documentation/api/metrics.md

```diff
@@ -0,0 +1,164 @@
+# Metrics - Get
+
+Gets a snapshot of metrics in the Prometheus exposition format of a single process.
+
+The metrics are collected from the following providers by default:
+- `System.Runtime`
+- `Microsoft.AspNetCore.Hosting`
+- `Grpc.AspNetCore.Server`
+
+All of the counters for each of these providers are collected by default.
+
+> **NOTE:** This route collects metrics only from a single process. If there are no processes or more than one process, the endpoint will not return information.
```

Comment:
I would also add a link to reverse server docs here as a means of constraining how many processes are visible to dotnet-monitor. This acts as a workaround to forcing one process.

Reply:
I'll add this once the reversed server docs are added.